spec/lib/utils/job-utils/pollers/ipmi-parser-spec.js

```diff
@@ -63,7 +63,7 @@ describe("ipmi-parser", function() {
                 //Make sure sdrType is either Discrete or Threshold
                 expect(sensor).to.have.property('sdrType')
                     .and.to.satisfy(function(val) {
-                        return val === 'Discrete' || val === 'Threshold';
+                        return val === 'Discrete' || val === 'Threshold' || val === "Analog";
```

Comment:
Is it necessary to add "Analog" here, when no sample data about ipmitool 1.8.11 is added?